### ISSUE TITLE
[ACCESSIBILITY]  - Large text causing buttons to flash and go offscreen

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#502]
**Describe the accessibility issue**
After launching app for the first time, user decides to enlarge text using iOS Large Text display feature. When the user toggles back to IL App the screen is flashing the CTA buttons and user is unable to press them. 

**Steps to reproduce the behavior:**

1. Fresh install of app (in order to access onboarding screens)
2. User launches app
3. User toggles to iOS Accessibility settings and chooses to enlarge display text
4. User toggles back to IL App
5. IL has CTA's flashing off and on at bottom of screen, resulting in user getting stuck

**Expected behavior**
Once user enlarges text, then toggles to IL App, the app should resize font and show CTA's on viewport.

**Screenshots**
See videos attached:
https://user-images.githubusercontent.com/82892873/125476284-dcc64fb0-6b32-4373-a514-5c7bcc880788.mp4

https://user-images.githubusercontent.com/82892873/125476303-9649d62a-45b9-49d0-b2b6-54d3f2e19056.mp4

**Smartphone** (please complete the following information):
**Device**: iPhone XR
**OS**: iOS 14.4

**Additional info**
I wanted to also see how this behaved when starting with Large text from the very beginning step of app launch.   When I did this the result was slightly different in that it resized text, CTA was not bouncing on and off screen but rather was off screen to the point where user is unable to press it. See image below:
![Large text upon launch renders buttons inaccessible](https://user-images.githubusercontent.com/82892873/125476696-c9097c18-0700-46df-acbd-07cd8fc52775.png)

**Comment by acovher2** [comment]
Hello, 
It has been 1 month; are there any updates on this issue? 
Please provide any information you can. Thank you.

**Comment by TodorBachvarov** [comment]
I'm not able to reproduce this issue with the latest version. Could you check again, probably it's already fixed.

**Comment by acovher2** [comment]
What device are you using? Did you increase font size to largest using device's accessibility settings? I'm able to reproduce on both iOS and android in latest version 2.6.21.

**Comment by TodorBachvarov** [comment]
I've tried with both ios and android. Please try with 2.6.23.

**Comment by acovher2** [comment]
How do I get this version. It is not in my TestFlight?

**Comment by acovher2** [comment]
(my teammate informed me it is to be released soon so disregard last question, thanks)

**Comment by acovher2** [comment]
Verified as fixed in 2.6.28
